server/pulp/server/db/model/dispatch.py

```diff
@@ -221,6 +222,14 @@ def as_dict(self):
             'total_run_count': self.total_run_count,
         }
 
+        for key, value in dict_repr.iteritems():
+            try:
+                dict_repr[key] = value.serializer(value).data
+            except AttributeError:
+                continue
```

Comment:
This is effectively `except AttributeError: pass`. I think I understand why it's find to just `continue` here, but I'd appreciate a comment about why missing keys are probably just fine.